package.json

```diff
@@ -12,13 +12,15 @@
   "dependencies": {
     "@apollo/client": "3.6.9",
     "@react-native-async-storage/async-storage": "1.17.7",
+    "@types/styled-components-react-native": "^5.1.3",
     "graphql": "16.5.0",
     "react": "17.0.2",
     "react-native": "0.66.4",
     "react-native-navigation": "7.23.1",
     "react-native-paper": "^4.12.3",
     "react-native-safe-area-context": "3.4.1",
-    "react-native-vector-icons": "8.1.0"
+    "react-native-vector-icons": "8.1.0",
+    "styled-components": "^5.3.5"
```

Comment:
É bom tirarmos esses `^`. Ele evita que a gente instale versões com potenciais vulnerabilidades ou ainda evita atualizações que possam quebrar compatibilidade.

Reply:
ainda sobre o `^`, tem duas formas pra sempre fixar a versão: 
1. quando for instalar um pacote, add a flag `save-exact`
```bash
npm install styled-components --save-exact
```
2. adicionar um arquivo, na pasta root do projeto, chamado `.npmrc`. esse arquivo terá somente essa linha 👇 :
```
save-exact=true
```
com esse arquivo, é só instalar o pacote normalmente
```bash
npm install styled-components
```
que o npm já vai instalar sem o `^` na versão. qualquer coisa chama no slack! 👍